docs/contributor-guide/developer-guide/deploy-system/prerequisites.md

```diff
@@ -0,0 +1,310 @@
+
+## Test AWS Authentication From Your Local Machine
+1. Test that your local machine can authenticate to the root user account
+    - Run `aws sts get-caller-identity --profile <name-of-parent-account>` in your terminal. Where `<name-of-parent-account>` is the name you specified in your `aws-cli` config file.
+    - The output should look something like:
+        ```
+        {
+            "UserId": "...",
+            "Account": "<root-account-number>",
+            "Arn": "arn:aws:iam::<root-account-number>:user/<your-user-name>"
+        }
+        ```
+
+2. Test that your dev account can authenticate with root user credentials on your local machine
+    - Run `aws sts get-caller-identity` without passing in the `--profile` flag. This will use the `default` profile set in your `aws-cli` config file which, if set correctly, should use the access credentials from the root account but with your dev user role.
+    - The output should look something like:
+        ```
+        {
+            "UserId": "...:botocore-session-...",
+            "Account": "<your-new-dev-account-number>",
+            "Arn": "arn:aws:sts::<your-new-dev-account-number>:assumed-role/OrganizationAccountAccessRole/botocore-session-..."
+        }
+        ```
+
+## Setup AWS Nuke
+AWS Nuke is a package that makes it easy to remove all resources from an AWS account. You will want to run this at the end of each working session (at least at the end of each day so we don't leave resources running overnight) and anytime you need to cleanup your AWS account to test new changes. It is destructive, so it's very important to set up your config file correctly to only clear out resources from your dev account. Check out the [README for the `aws-nuke` repository](https://github.com/rebuy-de/aws-nuke) for more information about the inner workings of `aws-nuke`.
+1. Download the correct binary for your machine's architecture/OS from the [aws-nuke repo](https://github.com/rebuy-de/aws-nuke/releases). Or on mac run `brew install aws-nuke` (go [here](https://brew.sh/) to install Homebrew if you don't already have it).
+2. If you downloaded the binary, unpack the tarball, rename the binary to `aws-nuke` and put it somewhere in your shell's path.
+3. Run `aws-nuke -h` to make sure it is installed correctly.
+4. In the root directory of your local `civiform-deploy` repo, create a `nuke.yaml` file and copy the [`nuke.yaml` file](https://github.com/civiform/cloud-deploy-infra/blob/main/e2e-test/nuke.yaml) used for end to end tests this file.
+5. Update the file with the correct blocklist and accounts (remove any references to e2e tests). For folks using the Exygy AWS account, remove any references to Google accounts. Take care to update the ACMCertificate value to the correct domain for your cert or the script will delete it every time. The config file will look different depending on the AWS root account you are using, but should end up looking something like:
```

Comment:
@nb1701 what do you think about these additions re: the `aws-nuke` config file? Does it feel clear enough what should change?

Reply:
Yep, LGTM!